ui/app/styles/components/services.scss

```diff
@@ -172,3 +172,21 @@ table.health-checks {
     }
   }
 }
+
+$tagPadding: 0.75em;
+
+.tag {
+  &.canary {
+    overflow: hidden;
+    &:before {
+      content: "Canary";
+      background-color: $blue-light;
+      color: $black;
+      line-height: 1.5em;
+      margin-left: -$tagPadding;
+      margin-right: $tagPadding;
+      padding: 0 $tagPadding;
+      align-self: normal;
+    }
+  }
+}
```

Comment:
nit-picking: would it be better to have this in [`tags.scss`](https://github.com/hashicorp/nomad/blob/main/ui/app/styles/core/tag.scss) and call the class `is-canary` to match others?